### PR TITLE
Javascript Problems | 13-04-2023 | 4 Problems

### PR DESCRIPTION
**Pull request by haraleaadi**

### CODE REVIEW

File: Problem2.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function checkEqual (arr){
```
Comment (ricky300300) [inline Problem2.js:1]: what is this program for? no question?

File: Problem3.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// for(let i=0; i<arr.length;i++)
+// {
+//     sum = sum+arr[i]
+// }
+// console.log(sum/arrLength)
+
+let arr = [5,3,6,4,2,7]
+let arrLength = arr.length;
+
+let sum = arr.reduce((acc,el)=>{
```
[the remark this answers is not in the document]
Comment (ricky300300) [reply Problem3.js:18]: you should make use of more functions instead of just writing all in a sequence

File: Problem4.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+let arr1=[33,78,67,-9,-110,34]
```
Comment (ricky300300) [inline Problem4.js:1]: where is the problem statement?

**Review by ricky300300 — CHANGES_REQUESTED** [review]
Please update as per comment and make sure to have multiple method/function to display solving the same thing in multiple ways

File: README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+        In JS today I learn some concepts and practice on it and use some concepts in js problems.
+        - foreach
+        - map
+        - filter
+        - reduce
+
+# React :
+
+     I've started to learn react from today.
```
Comment (nikhil-chougule) [inline README.md:11]: You can add these in description of PR and you can directly send an update to Ricky over slack

**Pull request merged**